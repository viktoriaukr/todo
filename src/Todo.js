import React from "react";

const Todo = ({ id, removeTodo, task }) => {
  const remove = () => removeTodo(id);
  return (
    <div>
      <li>
        {task}
        <button onClick={remove}>X</button>
      </li>
    </div>
  );
};

export default Todo;
