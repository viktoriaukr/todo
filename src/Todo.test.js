import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

test("render", () => {
  render(<Todo />);
});

test("matches snapshot", () => {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
