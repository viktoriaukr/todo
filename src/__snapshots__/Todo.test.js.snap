// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <li>
      <button>
        X
      </button>
    </li>
  </div>
</DocumentFragment>
`;
